### DISCUSSION TITLE
How to create a video sprite in v8

### DISCUSSION THREAD

**Discussion by Andiedie** [pixijs/pixijs#9912] [v8 General] [answered]
I try this code but failed
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <script src="https://unpkg.com/pixi.js@8.0.0-beta.11/dist/pixi.min.js"></script>
    <script>
      (async () => {
        const app = new PIXI.Application();
        await app.init();
        document.body.appendChild(app.canvas);

        const texture = await PIXI.Assets.load(
          "https://pixijs.com/assets/video.mp4"
        );

        const videoSprite = PIXI.Sprite.from(texture);

        app.stage.addChild(videoSprite);
      })();
    </script>
  </body>
</html>

```

Error: Uncaught DOMException: Failed to execute 'copyExternalImageToTexture' on 'GPUQueue': Failed to import texture from video element that doesn't have back resource.

<img width="429" alt="image" src="https://github.com/pixijs/pixijs/assets/21376471/d5bf585f-59c9-4052-ab12-8bf92fd05d9f">

**Comment by Zyie** [comment, marked as answer]
hey @Andiedie can you try this with the latest release? `8.0.0-rc`
I believe we fixed a couple of issues with video textures recently

**Reply by Andiedie** [reply]
I tried it on `8.0.0-rc`, the above code can work properly, thanks for the help.

By the way, will v8 version support working on node.js?

**Reply by Zyie** [reply]
the [node repo](https://github.com/pixijs/node) will need to be updated to support the latest API changes, but fundamentally the webgl renderer should still work.

we will be updating all the ecosystem repos probably after v8's final release

but if anyone wants to update the repo for us it should be a relatively straightforward task!
